src/features/Hours.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectHours } from '../stores/viewState';
+
+import styles from './Hours.module.css';
+
+const Hours = () => {
+  const hours = useSelector(selectHours);
+
+  if (hours === undefined) {
```

Comment:
really only undefined? I do `== null` nowadays

Reply:
Thanks for pointing. This check is no longer needed, because the check is happening on a higher tree level.